github/release_notes/model.py

```diff
@@ -71,7 +71,9 @@ class ReleaseNote:
     user_login: str
     is_current_repo: bool
     from_same_github_instance: str
-    cn_source_repo: str
+    current_component_name: str
+    source_component_name: str
+    source_component_access: gci.componentmodel.GithubAccess = field(compare=False)
```

Comment:
is it really required to omit it from comparison? however if it is: should we not also omit it from hash for reasons of symmetry?